NitroxClient/Debuggers/Drawer/Subnautica/UWEEventDrawer.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UWE;
+
+namespace NitroxClient.Debuggers.Drawer.Subnautica;
+
+public class UWEEventDrawer : IDrawer<Event<float>>, IDrawer<Event<PowerRelay>>
```

Comment:
Should be `UweEventDrawer` as it's 3+ letter acronym.

Reply:
I agree with this but I think we should be consistent. In this file we have GUI did we want to change that as well?

Reply:
If we could, yes I think so. But that's a Unity type.